common/src/main/scala/com/gu/media/model/PlutoIntegrationData.scala

```diff
@@ -79,6 +79,38 @@ object PlutoSyncMetadataMessage {
   }
 }
 
+case class PlutoResyncMetadataMessage(
+   `type`: String,
+    enabled: Boolean,
+    projectId: Option[String],
+    s3Key: String,
+    atomId: String,
+    title: String,
+    user: String,
```

Comment:
Do we need this property in this case class? Setting it to `system@video.gutools.co.uk` doesn't seem to have much value as we can switch on `type`. Removing `user` also makes the signature of `build` simpler as `email` can be removed.

Reply:
See above, pluto uses this to track which user put the thing in.  I'd rather have an explicit entry, so nothing needs to change on the other side, and that way these are conveniently logged with no code changes

Reply:
I've been through the code and it looks like a default value is used at the pluto end if this is not present so it should be safe to remove, i.e. it won't cause a crash.  However it is used in records of what has been imported, for tracking purposes, so `system@video.gutools.co.uk` makes sense at the pluto end as it's obvious from this record what triggered it compared to `unknown user`. I could hard-set it at the pluto end i suppose, but this would mean rejigging some of the logic because right now i can run exactly the same behaviour whether it's a new atom or a forced update.  Seems a bit bloaty to have to add an extra branch of code to fill something extra in at one end while changing the protocol when the same can be achieved just by setting a key at the other end

Reply:
Ah ok, yeah probably easier to send it in the message then. Given its just a string, I wonder if `media-atom-pluto-reindex` would be more descriptive and unique for logging/debugging purposes?